client/react/actions/clientActions.js

```diff
@@ -0,0 +1,7 @@
+import { Clients } from '../../../imports/api/clients/clients';
+
+export const getClientProfile = function getClientProfile(clientId) {
```

Comment:
Seems like this is not in use. What's the purpose of that code? If you need it for testing please move it to `/imports/__tests__/fixtures `